packages/cosmos_ui_components/lib/components/cosmos_circle_text_button.dart

```diff
@@ -26,13 +28,17 @@ class CosmosCircleTextButton extends StatelessWidget {
         onTap: onTap,
         child: Row(
           children: [
-            CircleAvatar(
-              // TODO: There needs to be a radius that matches the size in the design provided in Figma
-              maxRadius: 12,
-              minRadius: 12,
-              backgroundColor: isEnabled ? theme.colors.text : theme.colors.inactive,
-              foregroundColor: Colors.white,
-              child: Icon(icon, size: theme.fontSizeM),
+            SizedBox(
+              height: 24,
+              width: 24,
+              child: ClipRRect(
+                borderRadius: BorderRadius.circular(15),
```

Comment:
Wrapped the image around this because when giving color to image, it was appearing in a square shape. Do let me know if there is any better way of doing this

Reply:
Setting color to asset image will paint all pixels right. But using `colorBlendMode` will fix that. But that will result in putting that color blend to the whole container (which is a square). to prevent that I rounded the container around it.

Reply:
Fixed this now with latest PNGs being sent from the consumer end and applying simple color